src/problem_bank_scripts/problem_bank_scripts.py

```diff
@@ -30,6 +30,13 @@
 ## Loading files : https://stackoverflow.com/a/60687710
 import importlib.resources
 
+## learning outcomes
+topic_list = set()
+for subject in ["physics", "datascience", "stats"]:
```

Comment:
Since this function will be used by folks developing questions for specific subject question banks, I'd prefer to only load the CSV file for the appropriate bank (to enforce that the topic chosen is only from that bank, and not from another one). 

One idea to do this is as follows:

- get the current directory (can count on it to always be `instructor_subject_bank`) and split it twice:
    - first split on 'instructor_', grab the second result, and then split again on `_bank` and get the first result
    ```
    path = "/Users/firasm/Sync/EL/OPB/physics/instructor_physics_bank/source/003.Vectors/Vectors"
    subject = path.split('instructor_')[1].split('_bank')[0]
    ```
- Then load `pd.read_csv(f"{subject}.csv")`
- If there's an error, then load all the csv files (but print an error message saying there might be a problem with the topics)

Reply:
implemented [here](https://github.com/open-resources/problem_bank_scripts/pull/60/commits/e4e685b1e1cbf2f1f51b5c955f1593dcbaa16c33)